File: Codigos/Aula 4/Java/scr/model/Utilidades.java
package projeto.model;

import java.util.List;
import java.util.Random;

public class Utilidades {

    public static void popularLista(List<Integer> lista, long quantidadeNumeros, int inicio, int fim, boolean aleatorio) {
        Random gerador = new Random();
        
        if (aleatorio) {
            for (long i = 0; i < quantidadeNumeros; i++) {
                lista.add(gerador.nextInt(inicio, fim));
            }
        } else {
            for (long i = 0; i < quantidadeNumeros; i++) {
                lista.add((int) (inicio + i));
            }
        }
    }
}

File: Codigos/Aula 4/Python/scr/model/Utilidades.py
import random

class Utilidades:

    @staticmethod
    def popular_lista(lista: list[int], quantidade_numeros: int, inicio: int, fim: int, aleatorio: bool) -> None:
        if aleatorio:
            for _ in range(quantidade_numeros):
                lista.append(random.randint(inicio, fim - 1))
        else:
            for i in range(quantidade_numeros):
                lista.append(inicio + i)
